### ISSUE TITLE
Bug: gql import with typescript-react-apollo@2.0.0

### ISSUE THREAD

**Issue by nikolaik** [dotansimha/graphql-code-generator#4492]
**Describe the bug**

Tested `@graphql-codegen/typescript-react-apollo@2.0.0` to get latest apollo client v3 fixes. This introduced a bug in the generation of the gql import:
```typescript
import gql from 'Apollo.gql';
import * as Apollo from '@apollo/client';
/* <snip> */
export const UserDocument = gql`
/* <snip> */
```

**To Reproduce**

Visit https://codesandbox.io/s/small-sea-se072?file=/types.tsx

**Expected behavior**
```typescript
/* Note: No extra gql import since it is importable from `@apollo/client` */
import * as Apollo from '@apollo/client';
/* <snip> */
export const UserDocument = Apollo.gql`
/* <snip> */
```

**Comment by dotansimha** [comment]
Thank you for reporting it, working on a fix now :)

**Comment by dotansimha** [comment]
Fixed in 2.0.1 :)
